api/AudioListener.json

```diff
@@ -100,15 +105,15 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioListener/forwardX",
           "support": {
             "webview_android": {
-              "version_added": false,
+              "version_added": "52",
               "notes": "Supports a deprecated way of setting orientation — the <code>setOrientation()</code> method."
             },
             "chrome": {
-              "version_added": false,
+              "version_added": "52",
               "notes": "Supports a deprecated way of setting orientation — the <code>setOrientation()</code> method."
```

Comment:
Same as previous comment for all other such notes. I won't bother to mark all of these.